File: Taxi Booking.py
from typing import List


class Solution:
    def minimumTime(self, N : int, cur : int, pos : List[int], time : List[int]) -> int:
        mn=10**9
        for i in range(N):
            dist=abs(pos[i]-cur)
            mn=min(mn,dist*time[i])
        
        return mn

File: Taxi Booking.cpp
class Solution{
public:
    int minimumTime(int N,int cur,vector<int> &pos,vector<int> &time){
        int mn=1e9;
        for(int i=0;i<N;i++){
            int dist=abs(pos[i]-cur);
            mn=min(mn,dist*time[i]);
        }
        return mn;
    }
};
